var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -123,10 +125,12 @@ class Qt(Package):
     # QtQml
     depends_on("python", when='@5.7.0:', type='build')
 
-    # OpenGL hardware acceleration
-    depends_on("mesa", when='@4:+opengl')
```

Comment:
Sorry for confusing you, I think I misunderstood the original comment. I would keep it the way it is, including mesa. That is, just drop the last commit.

I think there is another PR which adds ‘gl’ virtual dependency that can be satisfied by Mesa or system provided libs.

Reply:
Ps. You would keep as it’s indeed not possible to combine it with version constraint


if sys.platform != 'darwin':
     depends_on("libxext", when='@3:4.99')

Reply:
Now I'm confused :confused: - do you want me to keep 05ce08de94 or revert it?

Reply:
Sorry for confusion, I would keep part of it (new dependency), and revert the rest.

Reply:
OK, please see c568d556a4d.